account.go

```diff
@@ -0,0 +1,22 @@
+package chartmogul
+
+const (
+	accountEndpoint = "account"
+)
+
+// Account details in ChartMogul
+type Account struct {
+	Name        string `json:"name,omitempty"`
+	Currency    string `json:"currency,omitempty"`
+	TimeZone    string `json:"time_zone,omitempty"`
+	WeekStartOn string `json:"week_start_on,omitempty"`
+
+	Errors Errors `json:"errors,omitempty"`
```

Comment:
Can there be errors as part of the returned JSON? I didn't notice on the other libraries.
It seems strange, because we don't submit any data for validation, so what validation errors would we receive?

Reply:
Errors struct for Errors - [Docs](https://pkg.go.dev/github.com/CloudNativeSDWAN/cnwan-reader/pkg/openapi#NewAPIResponseWithError)
```go
type Errors struct {
	// A list of errors occurred, if any.
	Errors []ResourceResponse `json:"errors,omitempty"`
}
```
This line is being used overall in our go library eg. customers, datasources, invoices.